spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigUtils.java

```diff
@@ -91,31 +100,143 @@ static String getApplicationNamespace(KubernetesClient client, String namespace,
 
 	}
 
-	/*
```

Comment:
these methods that I have added will need your eyes. I have moved away from reading individual secrets, but read a list instead, just like k8s-native does it. I have tried to add documentation everywhere and explain what I am doing.